### PR TITLE
Ci/1205782676034164 auto deploy

### PR DESCRIPTION
**Pull request by niazhussain**

### CODE REVIEW

File: .github/workflows/publishondockerhub.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        uses: appleboy/ssh-action@v1.0.0
+        with:
+          host: ${{ secrets.AUTOTEST_HOST }}
+          username: ${{ secrets.AUTOTEST_USERNAME }}
+          key: ${{ secrets.SSH_KEY_AUTOTEST }}
+          port: ${{ secrets.AUTOTEST_SSH_PORT }}
+          script: |
+            cd /home/jay/parachain-launch/yoyo/
+            echo ${{ steps.extract.outputs.tag }}
+            sudo TAG=${{ steps.image-version.outputs.tag }} docker compose down -v
```
Comment (sfffaaa) [inline .github/workflows/publishondockerhub.yml:63]: Will we fetch the docker image?

**Review by sfffaaa — CHANGES_REQUESTED** [review]
Please merge it to parachain_dev instead of dev branch

File: .github/workflows/publishondockerhub.yml (hunk 1 above)
Comment (niazhussain) [reply .github/workflows/publishondockerhub.yml:63]: yes, passing docker image version in TAG and  using TAG as arg in parachain-2000.Dockerfile

Comment (niazhussain) [reply .github/workflows/publishondockerhub.yml:63]: also changed base branch to parachain_dev

**Review by sfffaaa — APPROVED** [review]

**Pull request merged**